Exiled.Events/Patches/Events/Player/Transmitting.cs

```diff
@@ -56,20 +55,24 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                 });
 
             // -2 to return false
-            newInstructions[newInstructions.Count - 2].WithLabels(retLabel);
+            newInstructions[newInstructions.Count - 1].WithLabels(retLabel);
```

Comment:
-2?

Reply:
Copy paste.. and that was used in a different function so -2 made sense back then

Reply:
![image](https://user-images.githubusercontent.com/24619207/210819200-a27a5c26-dd1d-47e6-948e-f84830e21604.png)
 
 Interesting, github can't see change

Reply:
git moment

Reply:
😢